products/views.py

```diff
@@ -1,3 +1,62 @@
-from django.shortcuts import render
+import re
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Count
+
+from .models          import (
+    Product,
+    ProductGroup,
+    ProductGroupImage,
+    ProductGroupPackageType,
+    Relation
+)
+
+class ProductView(View):
+    def get(self, request, product_group_id):
+        try:
+            pgid = product_group_id
+
+            product_groups   = ProductGroup.objects.get(id = pgid)
```

Comment:
변수 선언시 단수, 복수를 잘 지켜주세요~
```suggestion
            product_group   = ProductGroup.objects.get(id = product_group_id)
```